SAKPaaS/src/app/components/search-bar/search-bar.component.ts

```diff
@@ -65,4 +65,9 @@ export class SearchBarComponent implements OnInit {
       this.searchService.triggerSearch(this.searchControl.value);
     }
   }
+
+  clearSearch(): void {
+    this.searchControl.setValue('');
+    this.triggerSearch();
```

Comment:
Ok, wenn du jetzt noch das `locations$ = this.locationsService.fetchLocations();` hinzufügst sollte es _wirklich_ gehen 😂

Reply:
Niemalsmerk, das funktioniert bei imr auch ohne. Kann sein, dass @patrickstecker mal seinen Cache putzen muss

Reply:
ja, man sollte auch testen, bevor man was macht :D